contracts/Vault.vy

```diff
@@ -581,6 +581,14 @@ def setWithdrawalQueue(queue: address[MAXIMUM_STRATEGIES]):
         if queue[i] == ZERO_ADDRESS and self.withdrawalQueue[i] == ZERO_ADDRESS:
             break
         assert self.strategies[queue[i]].activation > 0
+
+    for i in range(MAXIMUM_STRATEGIES):
```

Comment:
I agree now that this can be combined with above to reduce overhead

Reply:
updated.